### DISCUSSION TITLE
Face ID 成過去式？未來請「用心」解鎖 iPhone

### DISCUSSION THREAD

**Discussion by Tina-Peng-1566** [1101-GS4719-C/Python-Community-GS4719-C#2049] [General]
https://technews.tw/2024/08/01/heart-unlock-iphone/

日前蘋果公司通過以心跳解鎖的專利，使用者只須配戴Apple watch或手握iPhone(整個外殼都是心律測量設備)就能以心率感測驗證身分，解鎖所有Apple裝置，更獨特、方便及快速。

以前認為iPhone的Touch ID、Face ID已經很先進了，沒想到未來要「用心」解鎖!
若未來真有此項功能，我可能會選擇先觀望一陣子，畢竟是新技術，也不確定被盜用或保密程度如何，而且也因為更獨特，這算不算另類的隱私權?是否會有侵犯隱私的疑慮?或許可能還須制訂相關法律來規範公司的合理使用範圍等等。

**Comment by CtyceT** [comment]
回想過去Touch ID剛出時轟動全球，到後來各家智慧型手機品牌標配使用指紋解鎖設。現如今人臉解鎖都已經是隨處可見的生物辨識解鎖技術了。當初這些新技術的保密程度與隱私現在回看都有一定程度上的保證，因此我對於未來可能會出的心跳解鎖保持著樂觀的看法。

**Comment by kingsophy** [comment]
雖然我不是蘋果的用戶，但我很開心能夠看到這項大幅度的進步。我最在意的是法律規範的問題，隨著生物識別技術的不斷進步，確實需要更新相關法律和政策，以保護使用者的隱私權和數據安全。包括對生物識別數據的收集、使用和分享的明確規範等，用以確保企業不能濫用這些數據，來保障使用者的隱私。但我依舊樂觀看待這項功能的推出。

**Comment by ds0w0** [comment]
雖然我不是蘋果的用戶，但我對這項技術抱持懷疑。我不認為這會像想像中樂觀。確實，蘋果未來的心跳解鎖技術聽起來非常創新，也有可能為我們帶來更便捷的解鎖方式。但從目前的技術發展和隱私保護的角度來看，這也可能帶來一些挑戰和問題。

首先，雖然心跳解鎖可以提供獨特的安全性，但這樣的技術也可能會引發隱私擔憂。心跳數據與個人的生理特徵緊密相關，一旦這些數據被洩漏或濫用，可能會對用戶的隱私造成威脅。心率資料不僅能識別身份，還能揭示健康狀況，這使得數據保護成為一個更為複雜的問題。此外，這項新技術可能還會面臨實施上的困難。例如，如何確保心率數據的準確性和可靠性？在不同的環境條件下，心率感測的表現是否會穩定？這些都是需要考慮的重要問題。新技術的導入常常伴隨著許多未知的挑戰，可能需要一段時間來完善。

總體而言，雖然心跳解鎖技術看似具有潛力，但在廣泛實施之前，還需要謹慎考量其隱私影響和技術可靠性，也需要相應的法律來規範數據使用和保護。這樣才能真正確保技術能夠在提升便利性的同時，也保障用戶的隱私和安全。

**Comment by hunteryang0108** [comment]
謝謝你的分享，不過距離專利申請到投入商用應該還有很長的一條路要走，很可能不會那麼快見到這項技術問世。我對於這個心跳偵測技術的商用抱持懷疑態度，因為目前醫學專業領域普遍仍不承認Apple設備的心率監測，它能否作為健康管理、醫療設備使用仍待觀察。

**Comment by Jerryguo888** [comment]
蘋果的新技術要透過心跳解鎖，這件事確實創新且便利。雖然這種生物識別方法更獨特，但是否會引發隱私權的問題值得關注。如果未來推出這種功能，我可能會先看看其安全性與法規保障。

**Comment by Piyenparty** [comment]
雖然我是安卓的用戶，但看到科技這樣進步我是真的很開心，同時心裡其實也很希望真的可以看到安卓一些很有助益的科技進展，可以被我實際使用，那我會覺得我現在買這台手機沒有白買

**Comment by RU10171017** [comment]
哇第一次聽到可以「用心」解鎖手機，真的有夠未來感 😳 不過老實說我應該也會先觀望，畢竟心率這種生理資訊算蠻私密的，如果被不當使用感覺會比臉或指紋更敏感…
技術很酷，但安全跟隱私保障真的也要跟上才敢用 👀

**Comment by LINSHIHMIN** [comment]
可以用心跳解鎖是真的每個人心跳的聲音還有頻率等等都不太一樣嗎？其實我是抱著有點懷疑的態度在觀望，但已經申請專利的話，應該是真的有打算研究相關的內容，雖然很懷疑用心解鎖的功能，不過如果真的可以的話，我也蠻想試試看，但就是怕怕的畢竟手指跟面部解鎖都是外在的，突然要利用心臟有種奇怪的感覺。而且前面提到說會有隱私問題，甚至可能揭示個人的健康狀況，如果要用心解鎖，那這些心率數據必定會需要提供給apple，會不會有外洩的風險就又很難說了。

**Comment by wuyu06** [comment]
用心跳解鎖聽起來很酷，也讓人覺得科技又往前跨了一大步 
不過心跳這種生物特徵會不會被盜用或侵犯隱私，確實讓人有點擔心。像這種新技術，除了便利，也要有完善的安全和法律保障，才能讓大家放心使用

**Comment by mattwang1222** [comment]
以心率解鎖聽起來既科幻又便利，但我認為這樣高度個人化的辨識方式，更該重視資料的儲存與使用透明度。便利性固然吸引人，但若缺乏監督機制，反而可能讓我們「用心」換來的是更多隱私風險。

**Comment by edawdewv** [comment]
這個專利也太酷吧，比起Touch ID和Face ID更獨特、更個人化。不過我對這種新技術保持觀望態度。畢竟心跳資訊牽涉個人隱私，一旦被盜用可能造成嚴重後果，安全性和隱私保護絕對是關鍵。此外，這類生物特徵資料的使用，確實需要有明確的法律規範，避免濫用或侵犯使用者權益。總之，技術再先進，如何平衡便利性與隱私安全，才是未來的挑戰。
